test/cloud_watch_test.exs

```diff
@@ -10,7 +10,7 @@ defmodule CloudWatchTest do
   use ExUnit.Case, async: false
 
   setup_all do
-    {:ok, _} = Cycler.start_link()
```

Comment:
When I ran tests locally this was failing here or in log_stream_name_test depending which ran first since cycler doesn't exit after the test ends.